### PR TITLE
Preview font select

### PR DESCRIPTION
**Pull request by 70ray**

### CODE REVIEW

File: pinc/prefs_options.inc [hunk 1]
```diff
@@ -97,23 +97,51 @@ function get_user_proofreading_font($interface=NULL)
...
-    {
-        $full_font_family = $fallback_font;
-    }
-
-    $font_size_family = $font_size ? $font_size : 'unset';
+    return $full_font_family;
+}
 
-    return array($font_style, $font_size, $full_font_family, $font_size_family);
+function get_font_data_js()
```
[the remark this answers is not in the document]
Comment (70ray) [reply pinc/prefs_options.inc:141]: I think this should work. I'll try it.

[the remark this answers is not in the document]
Comment (70ray) [reply pinc/prefs_options.inc:141]: I think this should work. I'll try it.Done this now.

File: pinc/prefs_options.inc [hunk 2]
```diff
@@ -97,23 +97,51 @@ function get_user_proofreading_font($interface=NULL)
     }
     $font_size = array_get($proofreading_font_sizes, $font_size_i, '');
 
+    $full_font_family = get_full_font_families($interface)[$font_style_i];
+    $font_size_family = $font_size ? $font_size : 'unset';
+
+    return array($font_style, $font_size, $full_font_family, $font_size_family);
+}
+
+function get_font_styles($interface = NULL)
```
[the remark this answers is not in the document]
Comment (70ray) [reply pinc/prefs_options.inc:106]: Done this.

**Comment by cpeel** [comment]
Thanks Ray. We can probably update `userprefs.php` to use this too (eventually).

**Pull request merged**